content/pda.md

```diff
@@ -0,0 +1,523 @@
+# PDAs
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
+
+- Explain Program Derived Addresses (PDAs)
+- Explain various use cases of PDAs
+- Describe how PDAs are derived
+- Use PDA derivations to locate and retrieve data
+
+# TL;DR
+
+- A Program Derived Address (PDA) is derived from a program ID and an optional list of seeds.
+- PDAs are designed to be controlled by the program they are derived from
+- PDAs provide a deterministic way to find data stored on accounts
+- Programs can sign instruction for PDAs derived from its program ID
+
+## What is a Program Derived Address
+
+Program Derived Addresses (PDAs) are effectively account addresses designed to be signed for by a program rather than a secret key. As the name suggests, PDAs are derived using a program ID and an optional set of seeds. A PDA has the same form as a Solana public key, expect they are ensured to not be on the ed25519 curve and thus have no associated private key.
```

Comment:
@ZYJLiu sugg something along the lines of:
> Program Derived Addresses (PDAs) are account addresses designed to be signed for by a program rather than a secret key. As the name suggests, PDAs are derived using a program ID. Optionally, the derived accounts can also be found using a set of "seeds." More on this later, but these seeds will play an important role in how we use PDAs for data storage and retrieval. A PDA looks exactly like the public key of a Solana key pair.  There is an important difference, though - these public keys have no corresponding private key. In other words, unlike the public keys of Solana key pairs, PDAs are not on the ed25519 curve.